src/settings/LocationCampuses.js

```diff
@@ -5,9 +5,20 @@ import {
   injectIntl,
   intlShape,
 } from 'react-intl';
+
 import { ControlledVocab } from '@folio/stripes/smart-components';
 import { Select } from '@folio/stripes/components';
 
+const visibleFields = [
```

Comment:
There are changes in several files that looks like a code refactoring. Are you sure that it's a good idea to refactor and solve issue in one commit?

@VictorSoroka1 what do you think about it?

Reply:
Agree. This is what we have discussed to move in a new commit in order to prevent messing with it with implementation.

Reply:
OK guys. I did this for consistency with the LocationInstitutions component. I will consider this for the future.